Sprint-3/1-key-implement/3-get-card-value.js

```diff
@@ -8,7 +8,22 @@
 // write one test at a time, and make it pass, build your solution up methodically
 // just make one change at a time -- don't rush -- programmers are deep and careful thinkers
 function getCardValue(card) {
-    if (rank === "A") return 11;
+  let rank = card.charAt(0);
+  console.log(rank);
```

Comment:
This may have remained from when you were testing your function 🙂 - can you remove the console log?

Reply:
@delmorallopez  - console log (line 12) is still in your function. Please review.

Reply:
The console log is still in the file. It should be removed.